src/Compatibility/ControlGallery/src/Android/Compatibility.ControlGallery.Android.csproj

```diff
@@ -1,7 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="../../../../../eng/ControlGallery.Build.props" />
-  <Import Condition="'$(EnvironmentBuildPropsImported)' != 'True'" Project="..\..\..\..\..\eng\Environment.Build.props" />
```

Comment:
Do we want to delete this?

This is the props file that sets CI to true

https://github.com/dotnet/maui/blob/main/eng/Environment.Build.props#L7

Or is this already being imported?

Reply:
no you are right i just wanted to remove the ControLGallery.Build.props